server/sonar-server/src/main/java/org/sonar/server/issue/IssueQueryService.java

```diff
@@ -135,6 +137,20 @@ public IssueQuery createFromMap(Map<String, Object> params) {
     }
   }
 
+  private Date buildCreatedAfter(@Nullable Date createdAfter, @Nullable String createdSince) {
```

Comment:
I would use Preconditions.checkArgument(createdAfter == null || createdSince == null, "createdAfter and createdSince cannot be set simultaneously") at the beginning of the method